modules/web/src/app/shared/entity/settings.ts

```diff
@@ -41,6 +41,7 @@ export interface AdminSettings {
   userProjectsLimit: number;
   restrictProjectCreation: boolean;
   restrictProjectDeletion: boolean;
+  restrictProjectModification?: boolean;
```

Comment:
This isn't an optional field in API so it shouldn't be optional here either
```suggestion
  restrictProjectModification: boolean;
```